tasks/database-mysql.yml

```diff
@@ -18,6 +18,12 @@
     - python-mysqldb
   when: ansible_os_family == 'Debian'
 
+- name: Install the MySQL dependencies on FreeBSD
+  pkgng:
+    name: "mysql56-client"
```

Comment:
Is this the most stable name to depend on? Is there no number-free variant that will be stable over more years?

Reply:
Unfortunately I think yes. FreeBSD provides a separate port for both the server and the client portion of both MySQL and PostgreSQL. When building from source, you can specify a different version. 

I went with the version that is default in the source:
https://svnweb.freebsd.org/ports/head/Mk/bsd.default-versions.mk?view=markup

I guess one could extract it into a variable, so that all these dependencies could be included in the vars/ directory. This would have the benefit of helping those people who build their own packages.

Reply:
Do I understand correctly that the powerdns package does not depend on the mysql client package?

Reply:
It does have databases/mysql56-client as a dependency.
If you use the default packages that are built by the FreeBSD-team, it will be pulled in as a dependency while installing powerdns.
This would only install the client in the case where you "forgot" to install it the normal way, otherwise it would already find it there.

Reply:
I looked into this a little more, and it seems that there is also a similar problem with the python version, which comes in two flavors: py27- vs py36-
I will take another look at this.

Reply:
> If you use the default packages that are built by the FreeBSD-team, it will be pulled in as a dependency while installing powerdns.

Then we should think about *not* hardcoding `mysql56-client` in this role. If we remove it, the role will work for most users for many years. But if we keep it in, it might break on the next FreeBSD release.

Reply:
I checked this against a Fedora machine, where is also broke because it needed other packages. 
Since this would have created _yet another_ do-when block, I extracted it into a separate variable. 

Having it hardcoded in the vars/FreeBSD.yml file does not seem to much, the default is rarely changes. If someone builds their own packages, they can override it in the playbook.

Reply:
As per comment by RvdE(https://github.com/PowerDNS/pdns-ansible/pull/60#issuecomment-503032973), I removed the installation of dependencies for FreeBSD, but left them for the Linux flavours.